extensions/skins/conversation_v1/player.html

```diff
@@ -36,30 +34,22 @@
         cards, (b) the screen is wide, in which case both cards appear
         side-by-side.
       -->
-      <md-card class="conversation-skin-tutor-card conversation-skin-tutor-card-active"
-               ng-class="{'conversation-skin-left-card': isCurrentSupplementalCardNonempty() && canWindowFitTwoCards(), 'conversation-skin-has-subsidiary-cards': isCurrentSupplementalCardNonempty() && !canWindowFitTwoCards()}" ng-show="isPanelVisible(PANEL_TUTOR)">
-        <div class="conversation-skin-tutor-card-feedback-toggle-container" ng-if="activeCard.stateName && !isInPreviewMode">
-          <button class="conversation-skin-tutor-card-feedback-toggle"
```

Comment:
Oh, wait, I saw your PR message.

Are you sure you want to do this though? I ask because Ed is adding a 'speak' button to the cards as well. If we remove this, will there be space for that?